components/eamxx/src/share/io/tests/scorpio_interface_tests.cpp

```diff
@@ -20,18 +20,14 @@ TEST_CASE ("io_subsystem") {
 TEST_CASE ("write_and_read") {
   ekat::Comm comm (MPI_COMM_WORLD);
 
-  EKAT_REQUIRE_MSG (comm.size()<=4,
```

Comment:
These mods are just to allow running this test with an arbitrary number of ranks (rather than hardcoding 1-4).